portals/knowledge-management/src/Pages/Home.vue

```diff
@@ -145,6 +168,7 @@
                             <MagnifyingGlassIcon class="h-5 w-5 text-gray-400" aria-hidden="true" />
                         </div>
                         <input
+                            ref="globalSearchInput"
```

Comment:
@Orrison The example that you shared I am using that ref over here.

Reply:
@jayushi-canyon yeah I see now, I see that this is just a different way to do what I was thinking. There are so many different ways to do things in Vue lol